### ISSUE TITLE
Zooming out on TEdit 3.9

### ISSUE THREAD

**Issue by icedwarr** [TEdit/Terraria-Map-Editor#989]
My issue is that I cant use the scroll wheel to zoom out on TEdit 3.9, I'm on Windows 10.
Scroll wheel works on TEdit 3.10 (But I can't use that version, If I do any world that I edit and then save gets corrupted)

I just want to know if I am the only one with this problem. (I don't remember having this issue on Windows 8.1)

**Comment by enigmaquip** [comment]
Yes a changed in the latest windows 10 update broke zooming out in TEdit, it is fixed in the current version of TEdit, but not in older versions

What is happening with your world getting corrupted in 3.10? or are you using an older version of Terraria?

**Comment by icedwarr** [comment]
Yes I am using a older version of terraria (tmodloader) doesn't have the newest version of terraria working right now.
Also I don't want to play without tmodloader(for mods) and tedit I love em both.

But for now I just figured out a workaround: My mouse has a program that allows me to create macro's, So I made the scroll down function press "Ctrl+-" when pressed, Its a temp fix but it works pretty well.
